sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/SpecializedBlobClientBuilder.java

```diff
@@ -73,7 +74,7 @@
 
     private HttpClient httpClient;
     private final List<HttpPipelinePolicy> additionalPolicies = new ArrayList<>();
-    private HttpLogOptions logOptions = new HttpLogOptions();
+    private HttpLogOptions logOptions = getBlobDefaultLogOptions();
```

Comment:
You should just leave this as null. Create it only in the build method if it is non-null.

Reply:
Static will bring race condition, which two threads access the logOption, then one might overwrite the other. Prefer to call the method for every initialization.